src/Components/Presentational/RequestAssets/SnackBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
It might be helpful for users to have an X to dismiss the snack bar like the design spec. Similar to Gmail it will go away after 2 sec but you also have the option to close it right away.

Reply:
I talk to Scott about it. He said its nice to have but not necessary for this MVP.  But I did it anyways